### PR TITLE
Fix the wrong path to delete-x.png caused by typo

### LINKED ISSUE #90: Fix wrong path to `delete-x.png` caused by typo
`frotary` should be `frontary`. Fix the typo in `theme.css`

image

### PR DESCRIPTION
**Pull request by div-seungha**
Fix #90 

There is the wrong directory path `frotary`, which results in the browser console error. I fixed it correctly: `frontary`.

### CODE REVIEW

**Comment by sophie-cluml** [comment]
Please document the changes in the CHANGELOG.md

**Comment by div-seungha** [comment]
> Please document the changes in the CHANGELOG.md

I updated CHANGELOG.md, but I am not sure if I added the content correctly to our convention.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (sophie-cluml) [inline CHANGELOG.md:8]: 하단에 Unreleased에 대한 hyperlink 를 생성해주세요. 아래 내용을 적어주시면 됩니다.

```
[Unreleased]: https://github.com/aicers/frontary/compare/0.9.0...main   // 이 라인 추가
[0.9.0]: https://github.com/aicers/frontary/compare/0.8.2...0.9.0
```

Comment (div-seungha) [reply CHANGELOG.md:8]: 넵 추가하였습니다!

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
...
+
+### Fixed
+
+- Fixed the wrong path of a png file `/frotary` to `/frontary` in `theme.css`.
```
Comment (sophie-cluml) [inline CHANGELOG.md:12]:
```suggestion
- Fixed the wrong path of a delete-x.png file `/frotary` to `/frontary` in `theme.css`.
```

What do you think about making this more specific like above?

Comment (sophie-cluml) [reply CHANGELOG.md]: I think `/` should be removed, but, other things are simply suggestions. It's totally up to you.

Comment (div-seungha) [reply CHANGELOG.md]: The more specific, the better for all engineers. I've just fixed it as you figured out.

**Review by sophie-cluml — APPROVED** [review]
LGTM

**Pull request merged**